### PR TITLE
Making the world a better place...

### PR DESCRIPTION
**Pull request by mafergus**
*some stuff here*

### CODE REVIEW

**Comment by mafergus** [comment]
What does this do [DevAmp](https://devamp.io)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # DevAmpTesting
 Test repo for DevAmp
+
+AND SO IT BEGINS.
```
Comment (tylermann) [inline README.md]: wow, this looks really great... [DevAmp](https://devamp.io)

**Comment by tylermann** [comment]
💩 [DevAmp](https://devamp.io)

**Comment by tylermann** [comment]
this is just like the main chat for this pull request [DevAmp](https://devamp.io)

**Comment by mafergus** [comment]
😠 [DevAmp](https://devamp.io)

**Comment by mafergus** [comment]
Lol [DevAmp](https://devamp.io)

**Comment by tylermann** [comment]
if you click the github icon up to the left by the title, you can see that all of these messages are on the github pull request too [DevAmp](https://devamp.io)

File: README.md (hunk 1 above)
Comment (mafergus) [inline README.md]: Did you expect any less?? [DevAmp](https://devamp.io)

**Comment by mafergus** [comment]
Ah yeah, that icon isn't super obvious [DevAmp](https://devamp.io)

**Comment by mafergus** [comment]
To me at least [DevAmp](https://devamp.io)

**Comment by tylermann** [comment]
yeah... i don't really want people leaving >_< [DevAmp](https://devamp.io)

**Comment by mafergus** [comment]
Oh hey look at that [DevAmp](https://devamp.io)

**Comment by mafergus** [comment]
It's all in github [DevAmp](https://devamp.io)

**Comment by mafergus** [comment]
With the DevAmp icon haha awesome [DevAmp](https://devamp.io)

**Pull request merged**